lib/fs.js

```diff
@@ -82,8 +82,9 @@ function getOptions(options, defaultOptions) {
 }
 
 function copyObject(source) {
-  const target = {};
-  for (const key in source)
+  var target = {};
+  var key;
+  for (key in source)
```

Comment:
No, just the `const` usage. I thought there was an issue with using `var` in a for-in at one time, but I could be remembering wrong. I will change those.